packages/css/src/components/ordered-list/ordered-list.scss

```
@@ -36,6 +36,12 @@
   }
 }
 
+.amsterdam-ordered-list--inverse-color,
+.amsterdam-ordered-list--inverse-color:not(.amsterdam-ordered-list--no-markers),
```

Only a list without markers has a text colour.

```suggestion
.amsterdam-ordered-list--inverse-color:not(.amsterdam-ordered-list--no-markers),
```